modules/widgets/top-posts.php

```diff
@@ -319,6 +319,13 @@ function widget( $args, $instance ) {
 			return;
 		}
 
+		$layout = apply_filters('jetpack_top_posts_widget_layout', '', $posts, $display );
```

Comment:
Please add a document block for this filter. You can see examples of how it's done with other `apply_filters' calls, for example: https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L730